index.html

```diff
@@ -0,0 +1,153 @@
+	<title>Ryan Wilson's Portofolio</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href='style.css' type="text/css">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+</head>
+<body>
+	<div class="wrapper">
+		<header class="container-fluid">
+			<nav>
+				<ul id="main-nav" class="pull-left">
+					<li><a href="#">Home</a></li>
+					<li><a href="#skillsContent">Skills</a></li>
+					<li><a href="#projectsContent">Portfolio</a></li>
+					<li><a href="#">About</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+				<div class="pull-right quick-links">
+					<div class="header-name">
+						<span>ryan wilson</span>
+					</div>
+					<div class="header-social">
+						<a href="https://www.linkedin.com/in/ryanwilson801"><img src="images/linkedin.png"></a>
+						<a href="https://github.com/ryanadamwilson"><img src="images/github.png"></a>
+					</div>
+				</div>
+			</nav>
+		</header>
+		<div id="introContent">
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-xs-12">
+						<h1 id="title">Ryan Wilson | Front End Developer</h1>	
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-xs-12">
+						<p class="intro-text">I am a front end developer passionate about building modern and responsive websites.</p>
+						<div class="intro-port" onClick="portfolioBtn();">Check out some cool things I've built</div>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div id="skillsContent">
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-xs-12 col-md-6 pull-right skills">
+						<h2 class="contentHeader">Skills</h2>
+						<p>I create websites using HTML, CSS, and JavaScript. I most often prefer to work with WordPress, but am I also capable of using any content management system. I additionally focus on responsive design to ensure my sites work well on any screen size and device.</p>
+						<!--<p>I also offer copywriting, proofreading and editing as well as content management services.</p>
+						<p>My role in the creation of your web presence can be anything from a one-time meeting to an ongoing experience with continuous support.</p>-->
+						<p>My Key Skills:</p>
+						<ul id="skillsList">
+							<li>HTML</li>
+							<li>CSS 3</li>
+							<li>JavaScript</li>
+							<li>jQuery</li>
+							<li>WordPress</li>
+							<li>Bootstrap</li>
+							<li>Responsive Design</li>
+						</ul>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div id="projectsContent">
+			<div class="projects-module">
+				<div class="row">
+					<div class="col-sm-6 col-xs-12">
+						<div class="project">
+							<div class="col-xs-3">
+								<div class="circle-thumb fizzbuzz">Fizz</div>
+							</div>
+							<div class="col-xs-9">
+								<div class="project-content">
+									<div class="project-title">FizzBuzz JavaScript</div>
```

Comment:
This should be added dynamically to the page from your javascript, like you did with your fizzbuzz project. This may seem annoying because you only have a couple projects now, but when you get close to 10, you will be grateful you have a framework to plug the projects into. Checkout the demo to know what i mean.